jekyll/_cci2/version-control-system-integration-overview.adoc

```diff
@@ -84,7 +84,16 @@ include::../_includes/partials/tips/check-github-type.adoc[Check your GitHub int
 | [.circle-green]#**Yes**#
 | [.circle-red]#**No** ^1^#
 
-| xref:enable-checks#[GitHub Checks]
+| xref:status-updates#[Status Updates] 
+| [.circle-green]#**Yes**#
+| [.circle-green]#**Yes**#
+| [.circle-green]#**Yes**#
+| [.circle-green]#**Yes**#
+| [.circle-green]#**Yes**# ^4^
```

Comment:
Nitpicking... `[.circle-green]#**Yes**  ^4^#`, so the "4" has the same color as the "Yes"